cfme/utils/template/scvmm.py

```diff
@@ -11,55 +11,69 @@ class SCVMMTemplateUpload(ProviderTemplateUpload):
 
     @property
     def library(self):
-        return self.template_upload_data.get('vhds', None)
+        return self.template_upload_data["vhds"]
```

Comment:
**Question** why are we changing this?

Reply:
So that it will raise an error if that field is not in the yamls.

Reply:
Got it. so we are expecting that error to be raised.